javascript-conditionals/main.js

```diff
@@ -0,0 +1,115 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isUnderFive value:', isUnderFive(4));
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isEven value:', isEven(4));
+
+function startsWithJ(string) {
+  if (
+    string.startsWith('J') &&
+    string.charAt(0) === string.charAt(0).toUpperCase()
+  ) {
+    return true;
+  }
+}
+console.log('startsWithJ value:', startsWithJ('JavaScript'));
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const julie = {
+  name: 'Julie',
+  age: 27,
+};
+
+console.log('isOldEnoughToDrink value:', isOldEnoughToDrink(julie));
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const joolie = {
+  name: 'joolie',
+  age: 27,
+};
+
+console.log('isOldEnoughToDrive value:', isOldEnoughToDrive(joolie));
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+console.log('isOldEnoughToDrinkandDrive value:', isOldEnoughToDrinkAndDrive());
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 0 && pH <= 7) {
```

Comment:
This should be greater than or equal to 0

Reply:
![Screenshot 2023-07-13 at 11 40 17 AM](https://github.com/julieluangpraxay/c0723-code-solutions/assets/130415142/56e03128-822d-4af6-9d89-dd2e04da4ee3)